pysap/test/test_binding.py

```diff
@@ -424,8 +425,9 @@ def test_mr2d1d_init(self):
 
         mr = sp.MR2D1D()
         mr.transform(data)
-        assert(mr.cube is not None)
+        assert (mr.cube is not None)
 
+    @skip("Test fails, issue open to resolve.")
```

Comment:
I would rather use an xfail (exptected to fail [^1]) rather than silencing the test totally
[^1]: https://docs.pytest.org/en/6.2.x/skipping.html